site/content/docs/monitoring/check-results.md

```diff
@@ -8,36 +8,85 @@ menu:
 
 If you have checks running, you can select them on the main Checkly dashboard and get an overview of the results they have produced so far. 
 
-## Overview
+## Check results overview
 
-The Overview tab offers a summary with key metrics at a glance:
+Select a check and you will see a breakdown of its recent runs, together with key availability and performance metrics.
 
-![check results summary](/docs/images/monitoring/check-results-summary.png)
+![check results overview](/docs/images/monitoring/check-results-summary.png)
 
-1. **24-hour ratio**: the percentage of successful check runs over the last 24 hours
-2. **7-day ratio**: the percentage of successful check runs over the last 7 days
-3. **30-day ratio**: the percentage of successful check runs over the last 30 days
-4. **average**: average execution time across the last 24 hours
-5. **p95**: the [95th percentile](https://www.manageengine.com/network-monitoring/faq/95th-percentile-calculation.html) response time across the last 24 hours
-6. **p99**: the [99th percentile](https://en.wikipedia.org/wiki/Percentile) response time across the last 24 hours
+### Summary section
 
-You also have access to a graph showing status and response times per location across a timeframe of your choice. 
+The summary at the top of the page allows for filtering based all of the page's data points based on the selected timeframe and locations. 
+
+<img class="screenshot-partial" alt="check results overview summary" src="/docs/images/monitoring/check-overview-bar.png"/>
+
+Based on the user's selection, the metrics in the summary will also be updated to show the most important numbers at a glance. Single tabs in the summary can be clicked to skip to the related section of the page showing a more detailed breakdown of the relevant information.
+
+### Timeline
+
+Single check runs can be accessed by selecting them on the timeline and clicking View Details. A preview of the selected check run also appears just below the timeline.
```

Comment:
Single check runs can be accessed by selecting them on the time ribbon or by hovering dots and clicking View Details. View Details will open selected check run in new browser tab, simple click on time ribbon dot will show a preview of the selected check run just below the time ribbon.